### PR TITLE
Concept exercise review: lasagna

### PR DESCRIPTION
**Pull request by angelikatyborska**
I solved the lasagna exercise for the first time and noticed that the hints don't show up because of the wrong formatting of the file. I also went ahead and added some more obvious hints.

### CODE REVIEW

File: exercises/concept/lasagna/.docs/hints.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+- Use the [`times` arithmetic operator][arithmetics] to multiply the argument by `2`.
+
+## 2. Calculate the remaining oven time in minutes
+
+- Use the [`binary minus` arithmetic operator][arithmetics] to subtract the argument from `60`.
+
+## 3. Calculate the total working time in minutes
+
+- Reuse the `preptime` function defined in the first step.
+- Use the [`binary plus` arithmetic operator][arithmetics] to add the return value of `preptime` to the argument.
```
Comment (SaschaMann) [inline exercises/concept/lasagna/.docs/hints.md:18]:
```suggestion
- Use the [`plus` arithmetic operator][arithmetics] to add the return value of `preptime` to the argument.
```

File: exercises/concept/lasagna/.docs/hints.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+- You will need to define [functions][functions] and use [arithmetic operators][arithmetics].
+
+## 1. Calculate the preparation time in minutes
+
+- Use the [`times` arithmetic operator][arithmetics] to multiply the argument by `2`.
+
+## 2. Calculate the remaining oven time in minutes
+
+- Use the [`binary minus` arithmetic operator][arithmetics] to subtract the argument from `60`.
```
Comment (SaschaMann) [inline exercises/concept/lasagna/.docs/hints.md:13]:
```suggestion
- Use the [`minus` arithmetic operator][arithmetics] to subtract the argument from `60`.
```

**Review by SaschaMann — APPROVED** [review]
I feel like `binary` might cause some confusion if people expect this to be some kind of bit-manipulation operator based on that. Happy to leave it in though if you disagree

**Comment by angelikatyborska** [comment]
I copied the operator names exactly like they are on https://docs.julialang.org/en/v1/manual/mathematical-operations/#Arithmetic-Operators-1 but I guess a shorter name without "binary" will work too

**Comment by cmcaine** [comment]
Thanks!

**Pull request merged**